src/test/java/org/alfresco/repo/event2/AbstractContextAwareRepoEvent.java

```diff
@@ -264,7 +265,7 @@ protected NodeResource getNodeResourceBefore(RepoEvent<NodeResource> repoEvent)
      */
     protected void waitUntilNumOfEvents(int numOfEvents)
     {
-        await().atMost(5, SECONDS).until(() -> EVENT_CONTAINER.getEvents().size() == numOfEvents);
+        await().atMost(1000, SECONDS).until(() -> EVENT_CONTAINER.getEvents().size() == numOfEvents);
```

Comment:
Do we really need to make it 1000s? I think 5s or max 10s is more than enough. 
Also, the method JavaDoc needs updating if the wait time is not 5 second.

Reply:
I've forgot to change it back after debug, sorry.